src/controller/AutoCommissioner.h

```diff
@@ -51,6 +51,7 @@ class AutoCommissioner : public CommissioningDelegate
 
 private:
     DeviceProxy * GetDeviceProxyForStep(CommissioningStage nextStage);
+    void finalizePASESession();
```

Comment:
This needs to be documented, especially because it's not in any sense finalizing the PASE session....

Reply:
Changed name and added description:

    // Adjust the failsafe timer if CommissioningDelegate GetCASEFailsafeTimerSeconds is set
    void setCASEFailsafeTimerIfNeeded();